README.md

```diff
@@ -56,6 +56,10 @@ Windows is not tested and not supported for now.
 
 - Create a local virtual environment with `virtualenv .` and activate this with `source bin/activate`
 - Install the required dependencies with `pip install -r requirements.txt`
+- If `scancode-toolkit==2.0.1` requirements gives an error
+    - cd into `scancode-toolkit` directory and install `scancode-toolkit==2.0.1` with `python setup.py install`
+- Comment `pytest-django` in requirements.txt in case of error
+- Install the requests library with `pip install requests`
```

Comment:
These README change would not be needed if the requirements are correct. Could you instead provide a ptach for the requirements?

Reply:
ptach for requirements? Do you mean that I shall provide a pr for requirements?